README.md

```diff
@@ -31,6 +31,21 @@ because we find ourselves in a constant need to prioritize
 certain issues/PRs over others. If you think your issue/PR is very important,
 try to popularize it by getting other users to comment and share their point of view.
 
+## I want to file an issue!
+
+Great! We encourage you to participate in this open source project. We love Pull Requests, Bug Reports, ideas, (security) code reviews or any other kind of positive contribution.
+
+To make it easier to triage, we have these issue requirements:
+
+* Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean.
+* Every issue should have **exactly** one bug/feature request described in it. Please do not file meta feedback list tickets they are difficult to parse and address individual points.
```

Comment:
nit: I think the phrasing of "they are difficult to parse and address individual points" is a little ambiguous. Maybe "Please do not file meta feedback list tickets as it is difficult to parse them and address their individual points."?